profiling-ffi/src/exporter.rs

```diff
@@ -15,31 +15,37 @@ use std::borrow::Cow;
 use std::ptr::NonNull;
 use std::str::FromStr;
 
+#[allow(dead_code)]
```

Comment:
do we need both of these?

Reply:
Oops, forgot to remove the first one I tried. Thanks.